packaging/debian/package_tool/package_tool

```diff
@@ -3,41 +3,25 @@
 # Copyright (c) .NET Foundation and contributors. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-
-# Takes Parameters:
-#   $1 = Input Directory
-#   $2 = Output Directory
-#   $3 = Package Version
-
 set -e
-
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This tool is self-contained at the moment, without any dependency on the build scripts. I would prefer to keep it that way if possible, because this tool is intended to be somewhat independent. It's contract with our build being the directory conventions and debian_config.json